src/jdk.jpackage/windows/classes/jdk/jpackage/internal/WixFragmentBuilder.java

```diff
@@ -145,6 +145,25 @@ static boolean is64Bit() {
         return Architecture.is64bit();
     }
 
+    enum Arch {
+        X64,
+        X86,
+        AARCH64,
+        OTHER;
+    }
+
+    static Arch getArch() {
+        if (Architecture.isX64()) {
+            return Arch.X64;
+        } else if (Architecture.isX86()) {
+            return Arch.X86;
+        } else if (Architecture.isAARCH64()) {
+            return Arch.AARCH64;
+        } else {
+            return Arch.OTHER;
+        }
+    }
+
```

Comment:
isn't this enum definition redundant? Why not use `Architecture.current()` in WixPipeline's switch expressions?